movie-ranker/public/index.html

```
@@ -0,0 +1,723 @@
+      .movie-title { font-size: 1.2rem; }
+      .stats-bar .progress-wrap { display: none; }
+    }
+  </style>
+</head>
+<body>
+
+<header>
+  <span class="logo">🎬 Movie Ranker</span>
+  <div class="stats-bar" id="stats-bar"></div>
+  <nav>
+    <button id="nav-battle" class="active" onclick="switchView('battle')">⚔️ Battle</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)